### ISSUE TITLE
Move `getAppId()` and `getAppVersion()` somewhere else

### ISSUE THREAD

**Issue by baltpeter** [tweaselORG/appstraction#18]
Currently, they are on the platform API object:

https://github.com/tweaselORG/appstraction/blob/d07379a4a8c71e0d70d8b043fb23d600054d1ba6/src/index.ts#L142-L157

But that doesn't make much sense—they are completely independent of the configuration of those. They are essentially static utility functions.

Maybe we should thus move them somewhere else and just export them in `index.ts`?

**Comment by zner0L** [comment]
I agree. I didn't make much sense for me to put them there in the first place and it confused me for a while.

**Comment by baltpeter** [comment]
I'll make this into a `parseAppMeta()` function that returns `{ id: string; version: string }`. 

I can definitely see us parsing additional attributes in the future. And it doesn't make sense to have to parse the app multiple times to get both the version and app ID.
